platform/view/services/tracing/span.go

```diff
@@ -102,8 +102,8 @@ func newSpan(backingSpan trace.Span, labelNames []LabelName, operations metrics.
 }
 
 func defaultNow(t time.Time) time.Time {
-	if t.IsZero() {
-		return time.Now()
-	}
-	return t
+	//if t.IsZero() {
```

Comment:
forget something here?

Reply:
It is never the case so far, but yes I should fix this